### ISSUE TITLE
Fix blog build action

### ISSUE THREAD

**Issue by melange396** [cmu-delphi/www-main#955]
The "blog build" GH action was broken (by me w/ help from dependabot) somewhere seemingly between https://github.com/cmu-delphi/www-main/commit/dee269475bb6373c38048e360de032e3fd33e767 (the v3.2.12 release, which succeeded) and the commit of https://github.com/cmu-delphi/www-main/pull/940 , which failed.

A history of runs of the blog build GH Action can be found at https://github.com/cmu-delphi/www-main/actions/workflows/blog.yaml .  The first failure in the GH Actions history can be found at https://github.com/cmu-delphi/www-main/actions/runs/8344599900/job/22841412601 , and a snippet of the error output is below:
```
> Rscript -e "blogdown::build_site(local=FALSE, run_hugo=FALSE, build_rmd='md5sum')"

Rendering content/blog/2015-07-23-template-post.Rmd... Done.
Rendering content/blog/2020-08-10-hello-world.Rmd... Done.
Rendering content/blog/2020-08-26-fb-survey.Rmd... 
Quitting from lines 298-365 [unnamed-chunk-3] (2020-08-26-fb-survey.Rmd)
Error in `plot_theme()`:
! The `legend.pos` theme element is not defined in the element
  hierarchy.
Backtrace:
  1. base::eval(...)
  4. blogdown (local) `<fn>`(...)
  5. rmarkdown::render(...)
  6. knitr::knit(knit_input, knit_output, envir = envir, quiet = quiet)
  7. knitr:::process_file(text, output)
     ...
 27. knitr:::knit_print.default(x, ...)
 28. evaluate (local) normal_print(x)
 [30](https://github.com/cmu-delphi/www-main/actions/runs/8344599900/job/22841412601#step:9:31). ggplot2:::print.ggplot(x)
 32. ggplot2:::ggplot_gtable.ggplot_built(data)
 33. ggplot2:::plot_theme(plot)
Execution halted
Error: Failed to render content/blog/2020-08-26-fb-survey.Rmd
Execution halted
Error: Process completed with exit code 1.
```

**Comment by melange396** [comment]
looks like this may have been fixed on or before #949 somehow...  im gonna do some more checking before closing this.

**Comment by melange396** [comment]
looks good!
